bot/exts/evergreen/game.py

```diff
@@ -260,7 +260,7 @@ async def games(self, ctx: Context, amount: Optional[int] = 5, *, genre: Optiona
                 display_possibilities = "`, `".join(p[1] for p in possibilities)
                 await ctx.send(
                     f"Invalid genre `{genre}`. "
-                    f"{f'Maybe you meant `{display_possibilities}`?' if display_possibilities else ''}"
+                    f"Maybe you meant `{display_possibilities}`?" if display_possibilities else ''
```

Comment:
This approach does not produce the same result. The strings before the condition get implicitly joined, and the conditional then switches between 
```
f"Invalid genre `{genre}`. "
f"Maybe you meant `{display_possibilities}`?"
```
and the empty string instead of the previous behavior where it only displayed the invalid genre if there were no possibilities